openshift_day2_configurator/constants/ingress.py

```diff
@@ -0,0 +1,9 @@
+CREATE_NEW_INGRESS_CERTIFICATE: str = "Create new Ingress certificate"
```

Comment:
no need for a separate file; used in one module

Reply:
done